### ISSUE TITLE
t.getBoundingClientRect is not a function

### ISSUE THREAD

**Issue by nathanaelphilip** [callmecavs/jump.js#37]
I’m using Vue.js, and set this up in a method:

```
'navigate': function(target) {
   console.log(target);
   jump.jump(target,{duration: 500});
}
```

the target is just a selector: '#selector'

**Comment by callmecavs** [comment]
Here's the [MDN page](https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect).  Check the browser compatibility - its only IE9 plus (though I imagine Vue is as well).

Besides that, I've never used Vue, so I unfortunately can't help any further. This seems very specific to your use case/environment, so I'm going to close it. Good luck man.

**Comment by JasonStoltz** [comment]
I ran into the same issue. For me, it was happening as a result of running the package through the Webpack `UglifyJsPlugin`.

Specifically, it looked like after uglification it was hoisting all of the `top` calls [here](https://github.com/callmecavs/jump.js/blob/master/src/jump.js#L111) to the top of the function, before any of the case statements could be evaluated, the strings and `undefined`s were being passed to `top`, instead of a dom node like it expects.

I simply added `exlude: 'jump.js'`  to my UglifyJsPlugin config to resolve the issue.
